.github/ISSUE_TEMPLATE/usability.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Usability Improvement
+about: Make Bevy more fun to use!
+title: ''
+labels: C-Usability, S-Needs-Triage
+assignees: ''
+---
+
+## What inconsistency, frustration or unpleasant surprise did you encounter?
+
+Try to be as clear as you can.
+If this is part of a broader problem, include a specific example but feel free to point out other areas where this needs to be fixed.
+
+If you have relevant code snippets or screenshots to help others reproduce the problem, put them here.
+
+## How can we make this better?
+
+What would an ideal solution look like?
```

Comment:
I prefer "what would your ideal solution look like?" because it is more directed.